hack/make.sh

```diff
@@ -190,20 +190,19 @@ bundle() {
 }
 
 main() {
-	# We want this to fail if the bundles already exist and cannot be removed.
-	# This is to avoid mixing bundles from different versions of the code.
-	mkdir -p bundles
-	if [ -e "bundles/$VERSION" ] && [ -z "$KEEPBUNDLE" ]; then
-		echo "bundles/$VERSION already exists. Removing."
-		rm -fr "bundles/$VERSION" && mkdir "bundles/$VERSION" || exit 1
+	if [ -z "${KEEPBUNDLE-}" ]; then
+		echo "Removing bundles/"
+		rm -rf "bundles/*"
 		echo
 	fi
+	mkdir -p bundles
 
+	# Windows and symlinks don't get along well
 	if [ "$(go env GOHOSTOS)" != 'windows' ]; then
-		# Windows and symlinks don't get along well
-
 		rm -f bundles/latest
-		ln -s "$VERSION" bundles/latest
+		# preserve latest symlink for backward compatibility
+		ln -sf . bundles/latest
+		ln -sf . bundles/17.06.0-dev || true
```

Comment:
Instead of harcoding this could this be based off of $VERSION as well?

Reply:
The point of this PR is to ignore `VERSION`, so it seems backwards to keep depending on it.

We don't want to keep adding this symlink in the future. It's just here for backwards compatibility with the last version that  was supported. We should remove it after a few weeks once everyone has updated.

Reply:
I don't think anyone has hardcoded `17.06.0-dev` though for their build pipelines. Most (including Docker CE and EE) use `$VERSION` to denote where the bundles are located.

I'm fine with removing the `$VERSION` altogether but not fine with a hardcoded `17.06.0-dev`

Reply:
Ok, going to remove this version symlink